src/main/java/FC/Frame.java

```diff
@@ -1,2 +1,237 @@
+        studyButtonPanel.add(correctButton);
+        studyButtonPanel.add(incorrectButton);
+        studyButtonPanel.add(definitionButton);
+        studyButtonPanel.add(resetButton);
+
+        studyPanel.add(cardTextArea);
+        studyPanel.add(Box.createVerticalStrut(15));
+        studyPanel.add(studyButtonPanel);
+        studyPanel.setVisible(false);
+        //deckName.setHorizontalAlignment(SwingConstants.CENTER);
+
+        topPanel.add(deckName, BorderLayout.CENTER);
+        topPanel.add(numOfCards, BorderLayout.EAST);
+        topPanel.setVisible(false);
+
+        middlePanel.add(studyPanel);
+    }
+
+    private void setupNewDeckField() {
+        newDeckPanel = new JPanel();
+        newDeckPanel.setLayout(new BoxLayout(newDeckPanel, BoxLayout.Y_AXIS));
+        newDeckPanel.setBorder(new EmptyBorder(200,0,0,0));
+
+        newDeckLabel = new JLabel("Please Enter The Name Of Your New Deck");
+        deckNameTb = new JTextField();
+        enterDeckButton = new JButton("Enter New Deck!");
+        newDeckPanel.add(newDeckLabel);
+        newDeckPanel.add(Box.createVerticalStrut(15));
+        newDeckPanel.add(deckNameTb, Box.createVerticalStrut(15));
+        newDeckPanel.add(Box.createVerticalStrut(15));
+        newDeckPanel.add(enterDeckButton);
+        newDeckPanel.setVisible(false);
+        middlePanel.add(newDeckPanel);
+    }
+
+    private void newDeckClicked(){
+        newDeckPanel.setVisible(true);
+        existingDeckPanel.setVisible(false);
+        studyPanel.setVisible(false);
+        topPanel.setVisible(false);
+    }
+    private void comboboxClicked(){
+        deckSelected = deckBox.getSelectedItem().toString();
+        deckName.setText("Deck: " + deckSelected);
+        numOfCards.setText("");
+        newDeckPanel.setVisible(false);
+        existingDeckPanel.setVisible(true);
+        studyPanel.setVisible(false);
+        topPanel.setVisible(true);
+
+    }
+
+    private void studyOrResetClicked() throws SQLException {
+        studyController = new StudyController();
+        studyController.startNewStudySession(deckSelected);
+        currentCard = studyController.getNextToStudy();
+        cardTextArea.setText(currentCard.getTerm());
+        setNumOfCardsLabel();
+
+        correctButton.setEnabled(true);
+        incorrectButton.setEnabled(true);
+        definitionButton.setEnabled(true);
+        definitionButton.addActionListener(actionEvent -> cardTextArea.setText("<html>" + currentCard.getDef() + "</html>"));
+        correctButton.addActionListener(actionEvent -> correctButtonClicked());
+        incorrectButton.addActionListener(actionEvent -> incorrectButtonClicked());
+        studyPanel.setVisible(true);
+        topPanel.setVisible(true);
+    }
+
+    private void incorrectButtonClicked() {
+        currentCard = studyController.getNextToStudy();
+        cardTextArea.setText(currentCard.getTerm());
+    }
+
+    //BUG
+    private void correctButtonClicked() {
```

Comment:
BUG: 1. after resetting, re-enters this method even if not clicked.
2. when list reaches 0, user has to click correct again to disable buttons